plugins/installer/webinstaller/js/client.js

```diff
@@ -210,20 +210,16 @@ Joomla.apps.initialize = function() {
 	Joomla.apps.loaded = 1;
 	if (jQuery('#myTabContent').length) {
 		jQuery('<div id="appsloading"></div>')
-			.css("background", "rgba(255, 255, 255, .8) url('../media/jui/img/ajax-loader.gif') 50% 15% no-repeat")
 			.css("top", jQuery('#myTabContent').position().top - jQuery(window).scrollTop())
 			.css("left", jQuery('#myTabContent').position().left - jQuery(window).scrollLeft())
 			.css("width", jQuery('#myTabContent').width())
 			.css("height", jQuery('#myTabContent').height())
-			.css("position", "fixed")
-			.css("z-index", "1000")
-			.css("opacity", "0.80")
-			.css("-ms-filter", "progid:DXImageTransform.Microsoft.Alpha(Opacity = 80)")
-			.css("filter", "alpha(opacity = 80)")
 			.appendTo('#myTabContent');
```

Comment:
Thanks @dgt41 
Tested:
It's not working like expected. Joomla loader is not hiding on other tabs.
![installer-joomla-loader](https://cloud.githubusercontent.com/assets/2002921/7791900/150d450c-02c6-11e5-9347-465f7c3f5d0f.png)

----
It seems that you missed to include `#web` in following code.

```
.appendTo('#myTabContent #web');
```